lte/gateway/c/session_manager/StatsPoller.cpp

```diff
@@ -13,17 +13,27 @@
 #include <stdint.h>
 #include <atomic>
 #include <memory>
+#include <vector>
 #include "StatsPoller.h"
-#define COOKIE 0
-#define COOKIE_MASK 0
+#define OVS_COOKIE_MATCH_ALL 0xffffffff
 
 namespace magma {
 
 void StatsPoller::start_loop(
     std::shared_ptr<magma::LocalEnforcer> local_enforcer,
     uint32_t loop_interval_seconds) {
+  // for now poll all shards by passing in an empty vector
+  // to poll by cookie pass in a vector of shard ids, where
+  // each element is a shard id that maps directly to a cookie
   while (true) {
-    local_enforcer->poll_stats_enforcer(COOKIE, COOKIE_MASK);
+    std::vector<int> shard_ids{};
```

Comment:
can you add the logic for using your shardtracker thing, you already have everything so lets just do it in this pr